modules/nf-core/gecco/convert/main.nf

```diff
@@ -0,0 +1,57 @@
+process GECCO_CONVERT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gecco:0.9.10--pyhdfd78af_0':
+        'biocontainers/gecco:0.9.10--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta ), path(clusters)
+    tuple val(meta2), path(gbk)
```

Comment:
As gbks and clusters belongs should belong to the same sample, why this is not like:
`tuple val(meta ), path(clusters), path(gbk)`